cves/CVE-2015-5351.yml

```diff
@@ -60,24 +80,26 @@ fixes_vcc_instructions: |
   Refer to our instructions on how to find a Git SHA from an SVN revision.
 fixes:
    - commit: 1b1ce375c54ae2a17da94981af4ea97687f4b698
-     note: SVN rev 1720652, from the Tomcat website.
+     note: Belongs to tomcat repo. SVN rev 1720652, from the Tomcat website.
    - commit: c8a8ef0b5bbccd6beb06fd737751595ea810f2b1
-     note: SVN rev 1720655, from the Tomcat website.
+     note: Belongs to tomcat repo. SVN rev 1720655, from the Tomcat website.
    - commit: 129279660d1e338f319a68982dbffdabdfda2fd9
-     note: SVN rev 1720658, from the Tomcat website.
+     note: Belongs to tomcat80 repo. SVN rev 1720658, from the Tomcat website.
    - commit: a81565e60a4ad716df419cd816122f50324c0594
-     note: SVN rev 1720660, from the Tomcat website.
+     note: Belongs to tomcat80 repo. SVN rev 1720660, from the Tomcat website.
    - commit: 422cb0843d9e26dcc454520da3fca9a05bed4348
      note: SVN rev 1720661, from the Tomcat website.
    - commit: afa1a54323fbf410430e182098f95a31bcfcfb00
      note: SVN rev 1720663, from the Tomcat website.
    - commit:
      note:
+     
 vccs:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: e86a7a42fe4d82f8cb1e8bbb271656b49acd4d0a
+    note: |
+        This commit creates the index page for both of web applications with 
+        vulnerabilities. These index pages are Java Server Pages, which keep 
+        track user session by default.
```

Comment:
Missing some grammar here, "which keep track user session by default". Consider adding "of the" or something along those lines.